.github/workflows/generate-desktop-targets.yml

```diff
@@ -157,3 +157,31 @@ jobs:
         with:
           name: desktop.tar.gz
           path: desktop.tar.gz
+
+  desktop-linux-arm64:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Harden Runner
+        uses: step-security/harden-runner@63c24ba6bd7ba022e95695ff85de572c04a18142 # v2.7.0
+        with:
+          egress-policy: audit
+
+      - name: Install Go
+        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
+        with:
+          go-version: ${{ vars.GO_VERSION }}
+
+      - name: Checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Generate desktop.tar.gz
+        run: |
+          FLEET_DESKTOP_VERSION=$FLEET_DESKTOP_VERSION \
+          make desktop-linux-arm64
+
+      - name: Upload desktop.tar.gz
```

Comment:
This will collide with the artifact generated in line 158. Let's add a prefix?(`desktop-arm64.tar.gz`)

Reply:
I've actually run into a larger issue, no arm64 github actions machines to build the docker image

Reply:
Nvm it looks like I don't need to run it on an docker image

Reply:
Ah right, the beta images are not available yet? (https://github.blog/changelog/2024-06-03-actions-arm-based-linux-and-windows-runners-are-now-in-public-beta/)